_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+new %>% ggplot(aes(x = reorder(Social, -Use), y=Use)) +
+  geom_line(aes(color = evo),size = 0.9) +
+  geom_point(data = pos, aes(fill=evo), shape=24, size=1.2, color = "grey") +
+  geom_point(data = neg, aes(fill=evo), shape=25, size=1.2, color = "grey") +
+  geom_point(data = same, shape=18, size=2, fill="grey", color="grey") +
+  scale_y_continuous(
+    name = NULL,
+    limits=c(0, 100),
+    breaks = c(20, 40, 60, 80, 100),
+    labels = c("20" = "20", "40" = "40", "60" = "60", "80" = "80", "100" = "100%")
+  ) +
+  scale_x_discrete(name = NULL) +
+  #scale_color_manual(values= colors) +
+  scale_color_gradient(low="red", high="green") +
+  geom_label(data = last, aes(label = Use), nudge_x = 0.3, label.size = 0, fill="transparent") +
+  theme(
+    axis.ticks.x.bottom = element_line(color = "lightgrey"),
+    panel.grid.major.y = element_line(colour = "lightgrey", linetype = "dotted"),
+    panel.grid.minor.y = element_line(colour = "grey", linetype = "dotted"),
+    panel.background = element_rect(fill = "transparent"),
+    legend.position = "none",
+    axis.text.x.bottom = element_text(color="grey64", 
+                                    family = "Franklin Gothic"),
+    axis.text.y.left = element_text(color="grey64", 
+                                    family = "Franklin Gothic", size=10),
+    axis.line.x.bottom = element_line(color="lightgrey"),
+    axis.ticks.length = unit(0, "pt"),
+    axis.ticks.length.x.bottom = unit(5, "pt"),
+    plot.caption = element_text(family = "Franklin Gothic", 
+                                color = "grey64", hjust = 0),
+    plot.caption.position = "plot",
+    
+    plot.subtitle = element_text(family = "Georgia", 
+                                 color = "grey64", hjust = 0.02),
+    plot.title = element_text(family = "Franklin Gothic Demi", hjust = 0, size= 18),
+    plot.title.position = "plot",
+    aspect.ratio = 1/1.2
+  ) +
+  labs(title = paste(" Growing share of Americans say that they use YouTube;\n", 
+"Facebook remains one of the most widely used online\n",
+"platforms among U.S. adults"),
+       subtitle = "% of U.S. adults who say they ever use ... ('18-'21)",
+       caption = paste("Note: Respondents who did not give an answer are not shown.",
+       "Temporal series start in 2018\nfor all Social Media except for",
+       "TikTok and Nextdoor (2021)",
+       "\nSource: Survey of U.S. adults conducted Jan. 25-Feb. 8, 2021.", 
+       '\n"Social media use in 2021"'))
+    
+  
+
+```
+The idea is to give more importance to the actual position and the latest evolution, using arrows pointing to the last trend and gradient colors to remark when it's positive or negative.
+
+We could also add the logos from each Social Network below the name, to improve the understanding of the graph for those readers who are not very familiar with Internet (please note that this is a draft version). 
+
+## Addendums
+
+### Addendum 1: entire R code
```

Comment:
Nah, no need, please remove it.

Reply:
Sorry Iñali, you mean the titles or the whole code ?

Reply:
The whole section.